components/atom/table/src/index.js

```diff
@@ -21,17 +21,24 @@ const AtomTable = ({
   foot,
   fullWidth,
   cellPadding,
-  borderBottom
+  borderBottom,
+  onClick
```

Comment:
Maybe we could call the prop `onClickRow`. So we can give more semantics. 

```suggestion
  onClickRow
```

Reply:
It's a good point. Thank you!